x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/translations.tsx

```diff
@@ -98,3 +98,10 @@ export const SOURCE = i18n.translate(
     defaultMessage: 'Source',
   }
 );
+
+export const RULE_PREVIEW = i18n.translate(
+  'xpack.securitySolution.detectionEngine.createRule.stepDefineRule.rulePreview',
```

Comment:
Nit: `RULE_PREVIEW_TITLE` and `stepDefineRule.rulePreviewTitle` as per the [guideline](https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/GUIDELINE.md#naming-convention).

Reply:
done